### PR TITLE
Replace onClick listeners with links

### PR DESCRIPTION
**Pull request by csillag**
Task: https://app.clickup.com/t/86936b2yv

### CODE REVIEW

**Review by lubej — COMMENTED** [review]
There is one more location, that need to be replaced, here https://github.com/oasisprotocol/dapp-wrose/blob/1f184ed65672fce1a81daa6421335df2015d82af/src/pages/ConnectWallet/index.tsx#L55

But otherwise looks good to me.

**Comment by lubej** [comment]
Also, can we get rid of underline:

image

**Comment by csillag** [comment]
> There is one more location, that need to be replaced, 

Fixed

**Comment by csillag** [comment]
> Also, can we get rid of underline:

Fixed.

File: src/components/Account/index.module.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
   border-radius: 12px;
   background: var(--white);
   padding: 0.7125rem;
+  text-display: none;
```
Comment (lubej) [inline src/components/Account/index.module.css]: I don't think this is a valid css property?

Comment (csillag) [reply src/components/Account/index.module.css]: Oh right. It was supposed to be `text-decoration`. Fixed.

**Review by lubej — APPROVED** [review]

**Pull request merged**